transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/CreateTransferFragment.java

```diff
@@ -129,19 +136,22 @@ public void onClick(View v) {
 
         // transfer destination
         mTransferDestination = view.findViewById(R.id.transfer_destination);
-        mTransferDestination.setOnClickListener(new View.OnClickListener() {
+        mTransferDestination.setOnClickListener(new OneClickListener() {
             @Override
-            public void onClick(View v) {
-                //TODO temporary action for transfer destination clicked; it should open up transfer method list UI
-                Snackbar.make(mTransferDestination, "LIST Transfer clicked", Snackbar.LENGTH_SHORT).show();
+            public void onOneClick(View v) {
+                HyperwalletTransferMethod activeDestination =
```

Comment:
The only reason we need the current destination is for the token comparison in the adapter, isn't it? If that's the case, I'd rather to pass the token and not the whole object, then we can avoid the whole parcelable serialization of the whole object